docker_tasks/vector_ingest/handler.py

```diff
@@ -145,11 +144,35 @@ def load_to_featuresdb(filename: str, collection: str):
                 check=True,
                 capture_output=True,
             )
-            alter_datetime_add_indexes(collection)
+        elif collection in [
+            "lf_nfplist",
+            "lf_newfirepix",
+            "lf_fireline",
+            "lf_perimeter",
+        ]:
+            subprocess.run(
+                [
+                    "ogr2ogr",
+                    "-f",
+                    "PostgreSQL",
+                    connection,
+                    "-t_srs",
+                    "EPSG:4326",
+                    filename,
+                    "-nln",
+                    f"eis_fire_{collection}",
+                    "-overwrite",
+                    "-progress",
+                ],
+                check=True,
+                capture_output=True,
+            )
         else:
             print("Not a valid fireline collection")
             return {"status": "failure"}
 
+        alter_datetime_add_indexes(collection)
```

Comment:
If you you get to else condition (i.e. "Not a valid fireline collection"), then the `alter_datetime_add_indexes` will fail since it's not passing a collection

Reply:
line 172 has the `return` there that short circuits this right?

Reply:
Yes you are correct!